code/modules/client/client_defines.dm

```diff
@@ -42,7 +42,7 @@
 	var/played = 0
 	var/midi_silenced = 0
 	var/datum/soundOutput/soundOutput
-	var/list/volume_preferences = list(1, 0.5, 1, 0.85)//Game, music, admin midis, lobby music
+	var/list/volume_preferences = list(1, 0.5, 1, 0.7)//Game, music, admin midis, lobby music
```

Comment:
I am a fellow lobby music enthusiast but uuuh... it's *really* loud. Can we go down to like... half?

Reply:
It's not a linear scale AFAIK so half is a lot quieter, but we could further decrease it at 0.6 or 0.5

Keep in mind also not all tracks have the same volume, we could try reducing the audio volume of the outlier files directly

Reply:
```suggestion
	var/list/volume_preferences = list(1, 0.5, 1, 0.6)//Game, music, admin midis, lobby music
```